webhooks/cluster_test.go

```diff
@@ -40,14 +39,12 @@ func TestClusterDefaultNamespaces(t *testing.T) {
 			ControlPlaneRef:   &corev1.ObjectReference{},
 		},
 	}
-
 	webhook := &Cluster{}
-	t.Run("for Cluster", customDefaultValidateTest(ctx, in, webhook))
+	t.Run("for Cluster", customDefaultValidateTest(ctx, c, webhook))
+	c.Default()
```

Comment:
```suggestion
	g.Expect(webhook.Default(ctx,c)).To(Succeed())
```
I think we should keep testing the new default func here

Reply:
Similar in `TestClusterDefaultTopologyVersion`